db/seeds.rb

```diff
@@ -6,3 +6,28 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+   
+    teachers = Teacher.create!([
```

Comment:
Unfortunately this will throw when the seeds will be run more than once 😅 

Maybe [this blog post](https://mikechilson.com/blogs/seeding-your-database-in-rails-part-2) will be helpful to you :)

Reply:
After the first commit where teachers were created separately, I came up with the shorter code - the last commit. Please tell be which one is better.